draft-ietf-tls-tls13.md

```diff
@@ -1010,17 +1010,14 @@ of {{RFC5116}}. The key is either the client_write_key or the server_write_key.
 
 %%% Record Layer
        struct {
-           ContentType type;
```

Comment:
I thought we had agreed to have a dummy type outside the record, so that it doesn't break everyone's parser.